test/tabbable.test.js

```diff
@@ -13,18 +13,36 @@ var fixtures = {
 
 var fixtureRoots = [];
 
-function fixture(fixtureName) {
+function _createRootNode(doc, fixtureName) {
   var html = fixtures[fixtureName];
-  var root = document.createElement('div');
+  var root = doc.createElement('div');
   root.innerHTML = html;
-  document.body.appendChild(root);
+  doc.body.appendChild(root);
+  return root;
+}
+
+function _getTabbableIds(node) {
```

Comment:
I don't think there's any need for this leading underscore convention, because none of these functions are part of a public API — right?

Reply:
No problem, I'm happy to remove the underscores.